plugins/accelerated-peft/src/fms_acceleration_peft/gptqmodel/models/base.py

```diff
@@ -1105,45 +1109,50 @@ def skip(*args, **kwargs):
             # prepares the model on gpu in `trainer.train` to avoid unnecessary gpu usage
             device_map = {"": "cpu"}
 
-        load_checkpoint_in_model = False
-        # compat: runtime convert checkpoint gptq(v1) to gptq_v2 format
-        if quantize_config.format == FORMAT.GPTQ:
-            accelerate.load_checkpoint_in_model(
-                model,
-                dtype=torch_dtype,
-                # This is very hacky but works due to https://github.com/huggingface/accelerate/blob/bd72a5f1a80d5146554458823f8aeda0a9db5297/src/accelerate/utils/modeling.py#L292
-                checkpoint=model_save_name,
-                device_map=device_map,
-            )
-            # validate sym=False v1 loading needs to be protected for models produced with new v2 format codebase
-            if (
-                not quantize_config.sym
-                and not quantize_config.is_quantized_or_packed_by_v2()
-            ):
-                raise ValueError(
-                    f"Loading of a sym=False model with format={FORMAT.GPTQ} is only supported if produced by gptqmodel version >= {MIN_VERSION_WITH_V2}"
+        # low_cpu_mem_usage fix by flim@sg.ibm.com
+        # - load the checkpoint only if not low_cpu_mem_usage
+        # - or if low_cpu_mem_usage then only in the rank_0
+        if not low_cpu_mem_usage or is_local_dist_rank_0():
```

Comment:
`low_cpu_mem_usage` is default in fms-tuning because it is already set in our accelerate defaults, I [clarified that in the same issue ](https://github.com/foundation-model-stack/fms-hf-tuning/issues/12#issuecomment-2418311503)

And yes, prior to this issue, we had disabled `low_cpu_mem_mode` in the quant models, because in `transformers < 0.45`, it has always been the case